pkg/operator/terminationobserver/termination_observer.go

```diff
@@ -0,0 +1,272 @@
+package terminationobserver
```

Comment:
rename file to observer.go or even controller.go

Reply:
same comment as above